client/src/components/buttons/Button.tsx

```diff
@@ -209,35 +209,33 @@ type InlineSubmitButtonProps = {
  * @param {string} props.onSubmit function when click button
  */
 function InlineSubmitButton({
+  className,
+  doneText,
   id,
-  isSubmitting,
   isDone,
+  isMainButton,
   isReadOnly,
-  doneText,
-  text,
+  isSubmitting,
   onSubmit,
   pristine,
+  submittingText,
+  text,
   tooltipPristine,
-  className,
-  isMainButton,
 }: InlineSubmitButtonProps) {
   if (isDone) return <SuccessLabel text={doneText} />;
-  if (isSubmitting) {
+  if (isSubmitting)
     return (
-      <div className="inlineSubmit">
-        {" "}
-        <Loader className="mx-1" inline size={16} />
-      </div>
+      <LoadingLabel className="feedback mx-1" text={submittingText || " "} />
```

Comment:
This should render fine, without adding a space character:

```suggestion
      <LoadingLabel className="feedback mx-1" text={submittingText} />
```